dataland-keycloak/dataland_theme/login/templates/login.ftl

```diff
@@ -77,15 +77,28 @@
 
             <div>
                 <#list social.providers as p>
-                    <div id="social-${p.alias}">
-                        <button
-                            label="Join"
-                            class="p-button cursor-pointer w-full bg-white text-primary uppercase"
-                            onclick="location.href='${p.loginUrl}'"
-                        >
-                            LOG IN WITH ${p.displayName!}
-                        </button>
-                    </div>
+                    <#if p.alias="linkedin">
```

Comment:
Make LinkedIn picture appear in User Dropdown after login via LinkedIn

Reply:
Postponed due to much more complicated implementation than for Google Social Login